__tests__/index.test.ts

```diff
@@ -1136,7 +1139,8 @@ describe('Auth0', () => {
               refresh_token: TEST_REFRESH_TOKEN,
               client_id: TEST_CLIENT_ID,
               grant_type: 'refresh_token',
-              redirect_uri: 'http://localhost'
+              redirect_uri: 'http://localhost',
+              timeout: 10000
```

Comment:
The internal call to the token endpoint takes this value in ms, where the public-facing option asks for seconds. So there's a conversion here.